plugins/osdn/registry.go

```diff
@@ -399,18 +303,18 @@ func (registry *Registry) DeleteNetNamespace(name string) error {
 }
 
 func (registry *Registry) GetServicesForNamespace(namespace string) ([]osdnapi.Service, error) {
-	services, _, err := registry.getServices(namespace)
+	services, err := registry.getServices(namespace)
 	return services, err
```

Comment:
with the removal of the startVersion argument this can just be `return registry.getServices(namespace)`, but then also, you could just move the code from getServices() into this function and have GetServices() call `GetServicesForNamespace(kapi.NamespaceAll)`